addons/account/models/account_move.py

```diff
@@ -3982,6 +3982,21 @@ def _notify_get_recipients_groups(self, msg_vals=None):
     def _get_report_base_filename(self):
         return self._get_move_display_name()
 
+    def _get_report_attachment_filename(self):
+        self.ensure_one()
+        if self.state == 'posted':
+            name = self.name or _('INV')
+        else:
+            name = self._get_report_base_filename()
+        return name.replace('/', '_') + '.pdf'
+
+    def _get_report_mail_attachment_filename(self):
+        self.ensure_one()
+        invoice_name = (self.name or '').replace('/', '_')
+        pre_suffix = _('Invoice')
+        post_suffix = _('_draft') if self.state == 'draft' else ''
+        return '%(pre)s_%(name)s%(post)s' % {'pre': pre_suffix, 'name': invoice_name, 'post': post_suffix}
```

Comment:
No, we have to use the translation function here. if the language is in RTL instead of LTR, the translation function will handle it properly:

```suggestion
        return _('Invoice_%(name)s%(post)s', name=invoice_name, post=post_suffix)
```